pom.xml

```diff
@@ -142,6 +142,8 @@
 
         <!-- JSON Schema Validator -->
         <medeia-validator-jackson.version>1.1.1</medeia-validator-jackson.version>
+        <org.everit.json.schema.version>1.12.1</org.everit.json.schema.version><!-- TODO unification -->
+		<jackson-datatype-json-org.version>2.10.3</jackson-datatype-json-org.version>
```

Comment:
Formatting?

Reply:
Yeah - it would be good for everyone to stop using tabs for indentation in pom.xml files.